sdk/slot-hashes/src/lib.rs

```diff
@@ -0,0 +1,112 @@
+//! A type to hold data for the [`SlotHashes` sysvar][sv].
+//!
+//! [sv]: https://docs.solanalabs.com/runtime/sysvars#slothashes
+//!
+//! The sysvar ID is declared in [`sysvar::slot_hashes`].
```

Comment:
I think we need to update this comment. Is the sysvar to be declared in this crate, or will it be in a separate crate?

Reply:
Yeah I think the Sysvar impl will need to stay with the other sysvar code i.e. not in this crate. Updated the comment